libpod/runtime_volume_common.go

```diff
@@ -151,9 +151,6 @@ func (r *Runtime) newVolume(ctx context.Context, noCreatePluginVolume bool, opti
 		if err := os.MkdirAll(volPathRoot, 0700); err != nil {
 			return nil, fmt.Errorf("creating volume directory %q: %w", volPathRoot, err)
 		}
-		if err := idtools.SafeChown(volPathRoot, volume.config.UID, volume.config.GID); err != nil {
```

Comment:
This isn't going to work, `volume.config.UID` and `volume.config.GID` are actual options that can be set during volume creation.

Should we stop defaulting them to the container's UID/GID instead, and make this happen if and only if they were deliberately set?

Reply:
I think the tests checks that case.
The problem was the higher level directories were being set incorrectly.  We just want them to default to the users uid running podman.

Reply:
BTW, the answer is yes they should only be set if the user sets them. but not sure how that info can be recorded,  I guess we could default the UID/GID to -1?